languages/rust/exercises/concept/tuples/.docs/instructions.md

```diff
@@ -0,0 +1,56 @@
+# Instructions
+
+In this exercise you'll explore tuples in Rust by building a kitchen calculator. Imagine you are following a sensible gingersnap cookie recipe but it uses American units for measuring ingredients. You are most familiar with metric units. Being an intrepid programmer, you want to automate that work by writing a kitchen calculator.
+
+Use the following chart in your program:
+
+| Unit to convert | volume | in millilitres (mL) |
+| --------------- | ------ | ------------------- |
+| mL              | 1      | 1                   |
+| US cup          | 1      | 240                 |
+| US fluid ounce  | 1      | 30                  |
+| US teaspoon     | 1      | 5                   |
+| US tablespoon   | 1      | 15                  |
+
+## 1. Extract the numeric component from a tuple describing volume
+
+- Given the following tuple `(Unit::USCup, 1)`, return the 2nd element.
+
+```rust
+get_volume((Unit::USCup, 1))
+// 1
+```
+
+## 2. Convert the tuple to millilitres
+
+- Given a volume-pair tuple, `(Unit::USCup, 2.5)`, convert the volume to millilitres using the conversion chart
```

Comment:
The task description is usually written as text, without the leading dash:

```suggestion
Given a volume-pair tuple, `(Unit::USCup, 2.5)`, convert the volume to millilitres using the conversion chart
```

Reply:
Instead of using a very concrete example in the description, you could make it a generic description and have the example speak for itself.

```suggestion
Implement the `to_millilitres` function that takes a unit/volume pair tuple as its parameter, and returns a new tuple with its volume converted to millilitres using the conversion chart.
```